sclack/components.py

```diff
@@ -564,13 +564,32 @@ def keypress(self, size, key):
         return super(MessagePrompt, self).keypress(size, key)
 
 class Profile(urwid.Text):
-    def __init__(self, name, is_online=False):
+    def __init__(self, name, is_online=False, is_snoozed=False):
+        self.name = name
+        self.is_snoozed = is_snoozed
+
         if is_online:
-            presence_icon = ('presence_active', ' {} '.format(get_icon('online')))
+            self.presence_icon = ('presence_active', ' {} '.format(get_icon('online')))
```

Comment:
Here, we should add snooze icon like:
```
if is_snoozed:
            self.presence_icon = ('presence_active', ' {} '.format(get_icon('snooze')))
        
```
I have checked https://nerdfonts.com/ and see we can use `"snooze": "\uF9B1",`

Reply:
so the presence icon should indicate snooze too? 🤔

Reply:
yes, it should be snooze icon.

Reply:
ok, then, I'll do it this evening